info.csv

```diff
@@ -74,4 +74,4 @@ Abhay Singh Dhauni,  16SCSE102003,  1613102001,  Android Development,   3,    4,
 Swaranjali Jugran ,17SCSE1077025,1713112016,MACHINE LEARNING,2nd,1-BA  ,"C,C++,python,Java(basics)","CODING , READING AND WRITING TECH ARTICLES ",NA,jugran.swaranjali99@gmail.com,https://github.com/Swaranjali99,NA,NA
 Aryendra Prakash Singh,18SCSE1010508,18021011736,Web Development,1,10,"Java,C,Python","WebD,VUI",NA,aryendraprakashsingh@gmail.com,https://github.com/prakasharyendra,https://www.linkedin.com/in/aryendra-singh-441734112,NA
 Vansh Varshney,18SCSE1180001,18021180001,ANDROID DEV,1st,1,py java c,full stack web dev,na,vanshvarsh@gmail.com,https://github.com/vanshvarsh24,https://www.linkedin.com/in/vansh-varshney-1bb1b0181,na
-Tanya Verma, 17SCSE101526, 1713101507, AI & ML, 2, 8, Python, AI & ML, -, msstanya.30899@gmail.com, https://github.com/tanya3007, https://linkedin.com/in/tanya-verma-6904a517a, -
```

Comment:
Why You removed someone else's entry?

Reply:
I didn't remove sir I just added a new one

Reply:
I added my entry again I didn't remove any

Reply:
it's clearly showing that you removed that entry!

Reply:
I'll check this tomorrow morning!